packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/zoe-upgrade/vat-status.mjs

```diff
@@ -0,0 +1,105 @@
+// @ts-check
+import processAmbient from 'process';
+import dbOpenAmbient from 'better-sqlite3';
+
+/**
+ * @file look up vat incarnation from kernel DB
+ * @see {main}
+ */
+
+const swingstorePath = '~/.agoric/data/agoric/swingstore.sqlite';
+
+/**
+ * SQL short-hand
+ *
+ * @param {import('better-sqlite3').Database} db
+ */
+const dbTool = db => {
+  const prepare = (strings, ...params) => {
+    const dml = strings.join(' ? ');
```

Comment:
Not sure why you need to insert extra whitespace beyond that which the template author used.

```suggestion
    const dml = strings.join('?');
```